cmd/helm/search_hub.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/gosuri/uitable"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"helm.sh/helm/internal/monocular"
+)
+
+const searchHubDesc = `
+Search the Helm Hub or an instance of Monocular for Helm charts.
+
+The Helm Hub provides a centralized search for publicly available distributed
+charts. It is maintained by the Helm project. It can be visited at
+https://hub.helm.sh
+
+Monocular is a web-based application that enables the search and discovery of
+charts from multiple Helm Chart repositories. It is the codebase that powers the
+Helm Hub. You can find it at https://github.com/helm/monocular
+`
+
+type searchHubOptions struct {
+	searchEndpoint string
+	maxColWidth    uint
+}
+
+func newSearchHubCmd(out io.Writer) *cobra.Command {
+	o := &searchHubOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "hub [keyword]",
+		Short: "search for a keyword in charts",
```

Comment:
should this get updated to differentiate from `helm search repo`?

Reply:
I meant to change those and forgot. Thanks for catching that.